apps/connect/package.json

```diff
@@ -20,8 +20,8 @@
     "preview": "cross-env VITE_APP_CLUSTER=mainnet vite preview",
     "format": "npm run prettier -- --write ./src",
     "ts:check": "tsc --project ./tsconfig.json --noEmit --skipLibCheck",
-    "test": "cross-env VITE_APP_CLUSTER=testnet jest --coverage",
-    "test:watch": "cross-env VITE_APP_CLUSTER=testnet jest --coverage --watchAll=true"
+    "test": "cross-env VITE_APP_CLUSTER=mainnet jest --coverage",
```

Comment:
this doesn't seem right

Reply:
Do the tests necessarily have to be tested on the testnet env? The change was made because one of the tests was failing due to a name change in the symbol

Reply:
misread. marking as resolved